src/main/sensors/compass.c

```diff
@@ -57,11 +58,12 @@
 magDev_t magDev;
 mag_t mag;                   // mag access functions
 
-PG_REGISTER_WITH_RESET_FN(compassConfig_t, compassConfig, PG_COMPASS_CONFIG, 1);
+PG_REGISTER_WITH_RESET_FN(compassConfig_t, compassConfig, PG_COMPASS_CONFIG, 2);
```

Comment:
What is this done for?

Reply:
as noted on the commit "update compass config":

"* Bump the compass config PG version
* Re-arrange the values so that all the uint16_t's are aligned.
* Move magZero so it appears earlier in the structure as it's unlikely
to be removed/modifed compared to some other values that follow it."

The uint16 alignment has produced better code-space-saving assembler output where this has been done to other structures, it made sense to @ledvinap and I to bump the version at this time.

Reply:
Can you please just follow the rules, like everybody else?

Reply:
ok, yes, good point about doing the version bumps and re-ordering just before the release.  I'll update this as you suggest and we can look at optimizations nearer the time of the next release.